conf/messages.en

```diff
@@ -361,15 +361,15 @@ declaration.originationCountry.question = Where did the goods start their journe
 declaration.originationCountry.empty = Enter the country the goods started their journey
 declaration.originationCountry.error = Country is incorrect
 
-declaration.destinationCountries.title = Countries of destination
-declaration.destinationCountries.countryOfDestination = What is the destination country?
-declaration.destinationCountries.countryOfDestination.hint = The country where the goods are being exported to.
-declaration.destinationCountries.countryOfDestination.error = Country of destination is incorrect
-declaration.destinationCountries.countryOfDestination.empty = Country of destination cannot be empty
+declaration.destinationCountry.title = Destination country
```

Comment:
`declaration.destination.country.title`

Reply:
I treating `destinationCountry` as one thing, this is the reason that I'm not changing it